deployment/CE02SHSP_Deploy.csv

```diff
@@ -314,12 +314,21 @@ deploymentNumber,Reference Designator,sensor.uid,startDateTime,stopDateTime,lat,
 35,CE02SHSP-SP001-09-PARADJ000,CGINS-PARADJ-00504,2024-04-11T19:40:00,2024-04-24T19:05:00,44.63555,-124.30147,CGMCE-02SHSP-00035,SKQ202405S,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
 35,CE02SHSP-SP001-06-SPKIRJ000,CGINS-SPKIRJ-00281,2024-04-11T19:40:00,2024-04-24T19:05:00,44.63555,-124.30147,CGMCE-02SHSP-00035,SKQ202405S,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
 35,CE02SHSP-SP001-02-VELPTJ000,CGINS-VELPTJ-12876,2024-04-11T19:40:00,2024-04-24T19:05:00,44.63555,-124.30147,CGMCE-02SHSP-00035,SKQ202405S,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
-36,CE02SHSP-SP001-00-SPPENG000,CGCON-CSPPCE-00007,2024-05-02T20:35:00,,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
-36,CE02SHSP-SP001-08-CTDPFJ000,CGINS-CTDPFJ-00346,2024-05-02T20:35:00,,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
-36,CE02SHSP-SP001-01-DOSTAJ000,CGINS-DOSTAJ-00672,2024-05-02T20:35:00,,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
-36,CE02SHSP-SP001-07-FLORTJ000,CGINS-FLORTJ-01207,2024-05-02T20:35:00,,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
-36,CE02SHSP-SP001-05-NUTNRJ000,CGINS-NUTNRJ-00870,2024-05-02T20:35:00,,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
-36,CE02SHSP-SP001-04-OPTAAJ000,CGINS-OPTAAJ-00283,2024-05-02T20:35:00,,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
-36,CE02SHSP-SP001-09-PARADJ000,CGINS-PARADJ-00504,2024-05-02T20:35:00,,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
-36,CE02SHSP-SP001-06-SPKIRJ000,CGINS-SPKIRJ-00281,2024-05-02T20:35:00,,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
-36,CE02SHSP-SP001-02-VELPTJ000,CGINS-VELPTJ-12876,2024-05-02T20:35:00,,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
+36,CE02SHSP-SP001-00-SPPENG000,CGCON-CSPPCE-00007,2024-05-02T20:35:00,2024-08-14T19:08:00,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
```

Comment:
The small boat quick look report says deployment 36 was recovered at 2024-08-14T19:44 and the CSPP worksheet for deployment 36 only has the recovery date in it.  Which is correct?

Reply:
19:44 is correct, it just wasn't showing on the Excel spreadsheet, it is corrected now.